components/transaction/transactionSlice.ts

```diff
@@ -42,26 +53,21 @@ export const transactionSlice = createSlice({
             state.loading = false
         },
     },
-})
-
-export const fetchTransactions =
-    (group: Group | null) => async (dispatch: AppDispatch, getState: AppGetState) => {
-        if (!group?.gid) return
-        dispatch(transactionSlice.actions.fetchTransactionsStarted())
-        try {
-            const response = await fetch(
-                `${getState().global.backendUrl}/api/group/${group.gid}/transaction`
-            )
-            const users = await response.json()
-            dispatch(transactionSlice.actions.fetchTransactionsSuccess(users))
-        } catch (e) {
-            dispatch(
-                transactionSlice.actions.fetchTransactionsError(e as Error)
-            )
-        }
+    extraReducers: (builder) => {
+        builder.addCase(fetchTransactions.pending, (state) => {
```

Comment:
these look good, but seems like the old action reducers described above are unused and can be removed now